examples/1App_ALAEF_LCGTEP/watts_exec.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2022-2025 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+"""
+This example demonstrates how to use WATTS to run an A-LEAF calculation.
+"""
+
+import watts
+from pathlib import Path
+import numpy as np
+import time
+
+params = watts.Parameters()
+
+
+reference_prices = {
+    2022: 0.62, 2023: 0.62, 2024: 0.62, 2025: 0.62, 2026: 0.62, 2027: 0.62,
+    2028: 0.63, 2029: 0.63, 2030: 0.63, 2031: 0.76, 2032: 0.76, 2033: 0.76,
+    2034: 0.76, 2035: 0.76, 2036: 0.77, 2037: 0.77, 2038: 0.77, 2039: 0.77,
+    2040: 0.77, 2041: 0.78, 2042: 0.78, 2043: 0.78, 2044: 0.78, 2045: 0.78,
+    2046: 0.79, 2047: 0.79, 2048: 0.79, 2049: 0.79, 2050: 0.79, 2051: 0.80,
+    2052: 0.80, 2053: 0.80, 2054: 0.80, 2055: 0.80, 2056: 0.81, 2057: 0.81,
+    2058: 0.81, 2059: 0.81, 2060: 0.81
+}
+
+
+# Set nuclear prices from reference and calculated growth
+starting_year = 2031
+starting_price = 0.76
+# Set nuclear prices from reference up to 2030
+for year in range(2022, starting_year):
+    params[f'fuel_{year}'] = reference_prices[year]
```

Comment:
in fact, you can make a list or directly use the dictionary and jinja will be able to interpret it.
I would do something like:
params['fuel_price'] = reference_prices
Then:
{%for key, value in fuel_price.items()%}   {{ value|e }}  {%endfor%}

Reply:
cool I will try that